src/netaddress.cpp

```diff
@@ -742,7 +931,7 @@ std::string CService::ToStringPort() const
 
 std::string CService::ToStringIPPort() const
 {
-    if (IsIPv4() || IsTor() || IsInternal()) {
+    if (IsIPv4() || IsTor() || IsI2P() || IsInternal()) {
```

Comment:
1046e17c `IsCJDNS()`...should CJDNS IP strings be in IPv4 dotted-quad or IPv6 colon-hexadecimal notation?

Reply:
To my understanding CJDNS addresses have the same representation as IPv6, just different meaning.